.php_cs

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
.php_cs.dist should, is a simple way to enforce code style, this one don't need to versioned indeed